lib/palindrome_check.rb

```diff
@@ -1,5 +1,44 @@
 # A method to check if the input string is a palindrome.
 # Return true if the string is a palindrome. Return false otherwise.
 def palindrome_check(my_phrase)
```

Comment:
I don't see any comments on the time and space complexity. Can you think through the time and space complexity of your algorithm?
Gentle reminder: Always explain what n stands for while explaining your time and space complexity. That along with your explanation for reasoning behind your answer, is the complete answer for the time and space complexity. In this case, n is the number of characters in the input string parameter.

Reply:
Thanks for the feedback Shruti this is very helpful -I pushed up a refactor on this and included my thoughts on T and S complexity following your feedback above. I'm not confident in in my answer on T complexity when while loops are nested within each other so if you have clarification on calculating those I appreciate it!